handler/src/main/java/com/networknt/handler/LightHttpHandler.java

```diff
@@ -43,24 +44,24 @@
     String ERROR_NOT_DEFINED = "ERR10042";
 
     // Handler can save errors and stack traces for auditing. Default: false
- 	String CONFIG_NAME = "handler";
- 	String AUDIT_ON_ERROR = "auditOnError";
- 	String AUDIT_STACK_TRACE = "auditStackTrace";
+    String CONFIG_NAME = "handler";
+    String AUDIT_ON_ERROR = "auditOnError";
+    String AUDIT_STACK_TRACE = "auditStackTrace";
+
+    HandlerConfig config = (HandlerConfig) Config.getInstance().getJsonObjectConfig(CONFIG_NAME, HandlerConfig.class);
+    boolean auditOnError = config != null ? config.getAuditOnError() : false;
+    boolean auditStackTrace = config != null ? config.getAuditStackTrace() : false;
 
-	HandlerConfig config = (HandlerConfig) Config.getInstance().getJsonObjectConfig(CONFIG_NAME, HandlerConfig.class);
- 	boolean auditOnError = config != null ? config.getAuditOnError() : false;
- 	boolean auditStackTrace = config != null ? config.getAuditStackTrace() : false;
-    
     /**
      * This method is used to construct a standard error status in JSON format from an error code.
      *
      * @param exchange HttpServerExchange
-     * @param code error code
-     * @param args arguments for error description
+     * @param code     error code
```

Comment:
no need for tab here as per javadoc standard

Reply:
This is for align with the HttpServerExchange
Based on the example on Oracle doc https://www.oracle.com/ca-en/technical-resources/articles/java/javadoc-tool.html#format
You can see the "url" "name " in this example is also aligned.
I guess either aligned or not aligned is fine, what do you think?

Reply:
Ok I am fine if according to official documentation, it is acceptable. But i have not seen it much in practice.
Looks good to me.
Apprving